folium/plugins/beautify_icon.py

```diff
@@ -0,0 +1,94 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import (absolute_import, division, print_function)
+
+import json
+
+from branca.element import Figure, JavascriptLink, CssLink, MacroElement
+
+from six import iteritems
+
+from jinja2 import Template
+
+
+class BeautifyIcon(MacroElement):
+    """
+    Create a BeautifyIcon that can be added to a Marker
+    Parameters
+    ----------
+    icon: string
+        the Font-Awesome icon name to use to render the marker.
+    icon_shape: string
+        the icon shape
+    border_width: integer
+        the border width of the icon
+    border_color: string with hexadecimal RGB
+        the border color of the icon
+    text_color: string with hexadecimal RGB
+        the text color of the icon
+    background_color: string with hexadecimal RGB
+        the background color of the icon
+    inner_icon_style: string with css styles for the icon.
+        the css styles of the icon
+    spin: boolean
+        allow the icon to be spinning.
+    number: integer
+        the number of the icon.
+    Returns
+    -------
+    Icon names and HTML in obj.template_vars
```

Comment:
This class doesn't return anything, so the `Returns` part should be removed. (Technically it returns `None` but for a class docstring this is not documented.)